server/viame_tasks/tasks.py

```diff
@@ -217,4 +217,8 @@ def convert_images(self, folderId):
                 gc.delete(f"item/{item['_id']}")
                 count += 1
 
+    self.girder_client.addMetadataToFolder(
+        folderId, {"annotate": True},  # mark the parent folder as able to annotate.
+    )
+
```

Comment:
When running locally, I went to upload images that were in `.sgi` file format. I ran into the following error on the image conversion job:
```
TypeError: can only concatenate str (not "ObjectId") to str
```

Casting `folderId` to a `str` fixed the issue for me locally:

```suggestion
    self.girder_client.addMetadataToFolder(
        str(folderId),
        {"annotate": True},  # mark the parent folder as able to annotate.
    )

```